regression_test.py

```diff
@@ -11,13 +12,20 @@
 # 0 - script name
 # 1 - regression_test binary path
 # 2 - regression scenes dir
-# 3 - regression references dir
-# 4 - sofa build dir
+# 3 - sofa build dir
+# 4 (optional) - Refrence plugin directory. If not set, will take the parent dir of this file.
+
+
+os.system(sys.argv[1])
```

Comment:
This should be called at the end, after all env are set

Reply:
You are right. I'll do the change